lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Helpers/FB_PI_E621_SerialDriver.TcPOU

```diff
@@ -248,5 +248,21 @@ fbFormatString( sformat:=sErrMesg,
 	sOut => q_sResult);]]></ST>
       </Implementation>
     </Action>
+    <LineIds Name="FB_PI_E621_SerialDriver">
+      <LineId Id="3" Count="173" />
+      <LineId Id="2" Count="0" />
+    </LineIds>
+    <LineIds Name="FB_PI_E621_SerialDriver.a_ClearTrans">
+      <LineId Id="2" Count="13" />
+      <LineId Id="1" Count="0" />
+    </LineIds>
+    <LineIds Name="FB_PI_E621_SerialDriver.a_ErrorMesg">
+      <LineId Id="2" Count="3" />
+      <LineId Id="1" Count="0" />
+    </LineIds>
+    <LineIds Name="FB_PI_E621_SerialDriver.a_UnknownError">
+      <LineId Id="2" Count="4" />
+      <LineId Id="1" Count="0" />
+    </LineIds>
```

Comment:
You may have forgotten to set the separate line ids file option in one of your TwinCATs

Reply:
Yeah that's strange, I have Separate Line IDs set to True on both my local and prog node TwinCAT. I'm not sure how this got in here...

Reply:
Resolved in commit 725613d